love/hooks/use-questions.ts

```diff
@@ -29,6 +40,7 @@ export const useUserAnswersAndQuestions = (userId: string | undefined) => {
         setQuestions(questions)
       })
     }
-  }, [userId])
+  }, [userId, refresh])
```

Comment:
Move the logic currently within the useEffect into a function within the hook so that you can return a callback to it. Then call the function in the useEffect.